models/marts/core/fct_orders.sql

```diff
@@ -0,0 +1,31 @@
+with orders as (
+    select * from {{ ref('stg_orders') }}
+),
+
+payment as (
+    select * from {{ ref('stg_payments') }}
+),
+
+order_payments as (
+    select
+        order_id,
+        sum(case when status = 'success' then amount end) as amount
+
+    from payment
+    group by 1
+),
+
+final as (
+
+    select
+        orders.order_id,
+        orders.customer_id,
+        order_payments.amount
+

```

Comment:
if you bring through `order_date` here you should be able to do all the aggregations you want in `dim_customers` without importing 2 orders tables.